src/sagemaker/workflow/properties.py

```diff
@@ -98,17 +110,21 @@ def expr(self):
 class PropertiesList(Properties):
     """PropertiesList for use in workflow expressions."""
 
-    def __init__(self, path: str, shape_name: str = None, service_name: str = "sagemaker"):
+    def __init__(
+        self, step_name: str, path: str, shape_name: str = None, service_name: str = "sagemaker"
+    ):
         """Create a PropertiesList instance representing the given shape.
 
         Args:
             path (str): The parent path of the PropertiesList instance.
             shape_name (str): The botocore service model shape name.
             service_name (str): The botocore service name.
         """
-        super(PropertiesList, self).__init__(path, shape_name)
+        super(PropertiesList, self).__init__(step_name, path, shape_name)
+        self.step_name = step_name
         self.shape_name = shape_name
         self.service_name = service_name
+        self.path = path
```

Comment:
So the `self.path` is a relative path without the root: `f"Steps.{step_name}."` while the `self._path` (from super class) is an absolute path with the root right?
If so can we add some comments or update the attribute name to make it more clear?